app/__init__.py

```diff
@@ -176,3 +176,7 @@ def process_user_agent(user_agent_string):
         return "non-notify-user-agent"
     else:
         return "unknown"
+
+
+def cache_key_for_service_template_counter(service_id, limit_days=7):
+    return "{}-template-counter-limit-{}-days".format(service_id, limit_days)
```

Comment:
Is it worth having this in a separate file. Just thinking if we plan to add more cache keys `__init__.py` might not be the best place to have them.